src/thehexbot/Program.cs

```diff
@@ -195,11 +196,23 @@ private static void ProcessMention(IMention mention)
                 var attachment = File.ReadAllBytes(imageFile);
                 var media = Upload.UploadImage(attachment);
 
-                Tweet.PublishTweet($"@{mention.CreatedBy.ScreenName} Found {hex} in your tweet. Here you go!", new PublishTweetOptionalParameters
+                // replies to all the mentions in the tweet skipping the bot, because it will be tweet itself
+                string mentionedNames = "@"+mention.CreatedBy.ScreenName+" ";
+                foreach (var name in mention.UserMentions)
                 {
-                    InReplyToTweetId = mention.Id,
-                    Medias = new List<IMedia> { media }
-                });
+                    if (name.ScreenName != "thehexbot")
+                    {
+                        mentionedNames += "@"+name.ScreenName+" ";
+                        
+                    }
+
+                }
+
+                Tweet.PublishTweet($"{mentionedNames} Found {hex} in your tweet. Here you go!", new PublishTweetOptionalParameters
```

Comment:
I think the tweet look nicer if it says `Found '{hex}' ..." instead of just `Found {hex}`. The additional single quotes will add emphasis on which was the hex code.